lib/loader/index.js

```diff
@@ -135,6 +186,16 @@ module.exports = {
       var firstScript = getFirstScript();
 
       firstScript.parentNode.insertBefore(script, firstScript);
+
+      _loadedUrls[url] = true;
+
+      if (options.namespaceName) {
+        namespace.loadedUrls = _loadedUrls;
```

Comment:
Though @sirugh has a point, you should use namespace.registerProperty when first setting a property on the namespace, as the fact that this throws to prevent overwriting is important.